sound/soc/sof/intel/hda-loader.c

```diff
@@ -29,7 +29,7 @@ static int cl_stream_prepare(struct snd_sof_dev *sdev, unsigned int format,
 {
 	struct hdac_ext_stream *dsp_stream;
 	struct hdac_stream *hstream;
-	struct pci_dev *pci = to_pci_dev(sdev->dev);
+	struct pci_dev *pci = sdev->pci;
```

Comment:
Why do we need to re-introduce sdev->pci if we removed it before... You can find the info with the parent.